<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionAboWindow</class>
 <widget class="QMainWindow" name="gestionAboWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1138</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Retour">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>462</y>
      <width>75</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#Retour{
background-color: transparent;
border-image: url(:/img/img/retour.jpg);
background: none;
border: none;
background-repeat: none;
color : gray ;

font-size: 18px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_5{
background-color: transparent;
border-image: url(:/img/img/espace admin.jpg);
background: none;
border: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>761</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="lineEdit_prenom_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>160</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>109</x>
        <y>200</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Type pension</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nom_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>109</x>
        <y>240</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date début abonnement</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>109</x>
        <y>80</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>240</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>80</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="type_pension_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>200</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>pension complète</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>demi pension</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>demi pension + petit déjeuner</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>109</x>
        <y>120</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>109</x>
        <y>160</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prénom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>280</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>400</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Type pension</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>350</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>chercher par id </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>350</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>211</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>185</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prenom_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>380</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>180</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>370</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Prénom</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>50</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>trier</string>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>77</width>
         <height>54</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>trier par nom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_trier_id_2">
          <property name="text">
           <string>trier par id</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="modifier_etu_3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>470</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifer</string>
      </property>
     </widget>
     <widget class="QComboBox" name="lineEdit_type_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>410</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>pension complète</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>demi pension</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>demi pension + petit déjeuner</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit_chercher_id_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>350</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_afficherAbonne_par_type_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher ordonné par type</string>
      </property>
     </widget>
     <widget class="QTableView" name="tabetudiant_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>441</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_date_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>440</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_calculernbrepresence_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>calculer nombre presence total</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>440</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Date debut abonnement</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_afficher_nbre_par_type_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher nbre abonné par type</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nom_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>350</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_presence_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_ajouter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_ajouter{
background-color: transparent;
border-image: url(:/img/img/ajouter.jpg);
background: none;
border: none;
background-repeat: none;
color : gray ;

font-size: 18px;
}</string>
    </property>
    <property name="text">
     <string>ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_afficher">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_afficher{
background-color: transparent;
border-image: url(:/img/img/chercher.jpg);
background: none;
border: none;
background-repeat: none;
color : gray ;

font-size: 18px;
}</string>
    </property>
    <property name="text">
     <string>afficher</string>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>Retour</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>pushButton_ajouter</zorder>
   <zorder>pushButton_afficher</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1138</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
